--- zcash_protocol-0.10.5/src/consensus.rs
+++ vendor/zcash_protocol/src/consensus.rs
@@ -534,7 +534,7 @@
             NetworkUpgrade::Nu6_2 => Some(BlockHeight(4_052_000)),
             NetworkUpgrade::Nu6_3 => Some(BlockHeight(4_134_000)),
             #[cfg(zcash_unstable = "nu7")]
-            NetworkUpgrade::Nu7 => None,
+            NetworkUpgrade::Nu7 => Some(BlockHeight(4_465_026)),
         }
     }
 }
@@ -757,7 +757,7 @@
             0x5437_f330 => Ok(BranchId::Nu6_2),
             0x37a5_165b => Ok(BranchId::Nu6_3),
             #[cfg(zcash_unstable = "nu7")]
-            0xffff_ffff => Ok(BranchId::Nu7),
+            0x7719_0ad9 => Ok(BranchId::Nu7),
             _ => Err("Unknown consensus branch ID"),
         }
     }
@@ -778,7 +778,7 @@
             BranchId::Nu6_2 => 0x5437_f330,
             BranchId::Nu6_3 => 0x37a5_165b,
             #[cfg(zcash_unstable = "nu7")]
-            BranchId::Nu7 => 0xffff_ffff,
+            BranchId::Nu7 => 0x7719_0ad9,
         }
     }
 }
